hal/src/main/native/cpp/proto/ControlDataProto.cpp

```diff
@@ -94,16 +108,22 @@ std::optional<mrc::ControlData> wpi::Protobuf<mrc::ControlData>::Unpack(
 
 bool wpi::Protobuf<mrc::ControlData>::Pack(OutputStream& Stream,
                                            const mrc::ControlData& Value) {
-  std::string_view OpMode = Value.GetOpMode();
-  wpi::PackCallback OpModeCb{&OpMode};
+  std::string_view CurrentOpMode = Value.GetCurrentOpMode();
+  wpi::PackCallback CurrentOpModeCb{&CurrentOpMode};
+  std::string_view SelectedTeleopOpMode = Value.GetSelectedTeleopOpMode();
+  wpi::PackCallback SelectedTeleopOpModeCb{&SelectedTeleopOpMode};
+  std::string_view SelectedAutonOpMode = Value.GetSelectedAutonOpMode();
+  wpi::PackCallback SelectedAutonOpModeCb{&SelectedAutonOpMode};
```

Comment:
We should decide on naming.  I have a mix of short “Auto” and “Teleop” names and long “Autonomous” and “Teleoperated” names, probably don’t want to add “Auton” to the mix lol.

Reply:
Agreed. I don't even know where I got Auton and Teleop from.

Reply:
Fixed